src/app/services/data-grid.services.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@angular/core'
+import { HttpClient } from '@angular/common/http'
+import { Observable } from 'rxjs'
+import { Student } from '../models/student.models'
+
+@Injectable({
+	providedIn: 'root',
+})
+export class DataGridServices {
+	baseURL = 'http://localhost:5000/student'
+
+	constructor(private http: HttpClient) {}
+
+	getStudents() {
+		console.log('here')
+		return new Observable<Student[]>((observer) => {
+			this.http
+				.get<Student[]>('http://localhost:5000/student')
```

Comment:
`return this.http.get<Student[]>('http://localhost:5000/student')` is enough. move urls to env